docs/books/user-guide/routing.adoc

```diff
@@ -102,6 +102,17 @@ Link routing supports local transactions to a broker.
 +
 With a link route, consumers can provide server-side selectors for broker subscriptions.
 
+[id='considerations-for-broker-connection-failure-handling']
+=== Considerations for Broker Connection Failure Handling
+
+The message routing and link routing mechanisms both enable you to transfer messages to brokers. However, they differ in how they behave if the connection to the broker is lost. If you require the ability for the connection to be reestablished automatically, you should use message routing (with autolinks).
```

Comment:
This is a bit misleading because it suggests that with link-routing, the connection to a broker is not automatically reestablished.
In all cases, a connector will attempt to reconnect after failure.  This is true regardless of whether that connection is used with auto-links or link-routes.